composer.json

```diff
@@ -27,6 +27,7 @@
         "drupal/acquia_cms_starter": "dev-develop",
         "drupal/acquia_cms_toolbar": "dev-develop",
         "drupal/acquia_cms_tour": "dev-develop",
+        "drupal/consumer_image_styles": "^4.0",
```

Comment:
@chandan-singh7929 why do we need to include this module?

Reply:
This module is required to setup headless starterkit.